### PR TITLE
change docker.io/ibmcom to quay.io/ibmodffs

### PR DESCRIPTION
**Pull request by bvered**
Signed-off-by: 662962756 <bvered@il.ibm.com>

### CODE REVIEW

**Comment by bvered** [comment]
[change docker.io/ibmcom to quay.io/ibmodffs registry](https://trello.com/c/qyOLlVp3/289-change-dockerio-ibmcom-to-quayio-ibmodffs-registry)

File: .github/workflows/iso-ci.yaml [hunk 1]
```diff
@@ -35,15 +35,16 @@ jobs:
       run: |
         make test
 
-    - name: login to Docker Hub
+    - name: login to Quay Hub
```
Comment (tyichye19) [inline .github/workflows/iso-ci.yaml:38]: can you change this to `Quay` or `Quay.io`?  `Hub` is relevant only for Docker Hub

**Review by tyichye19 — APPROVED** [review]
LGTM

File: .github/workflows/iso-ci.yaml (hunk 1 above)
Comment (bvered) [reply .github/workflows/iso-ci.yaml]: fixed

**Pull request merged**